webroot/js/components/chat/chat-input.js

```diff
@@ -25,6 +25,7 @@ import {
   CHAT_CHAR_COUNT_BUFFER,
   CHAT_OK_KEYCODES,
   CHAT_KEY_MODIFIERS,
+  CHAT_MAX_MESSAGE_LENGTH,
```

Comment:
We shouldn't need this anymore

Reply:
I was seeing an exception thrown because it's used here:

https://github.com/owncast/owncast/blob/develop/webroot/js/components/chat/chat-input.js#L248

Reply:
Looks like line 252 needs to be updated so that it uses `props.inputMaxBytes`.

Reply:
Actually no. `inputText` is no longer needed.